doc/BuildingWithCMake.md

```diff
@@ -54,6 +54,104 @@ CMake can generate build files for a number of backends. You can select the gene
 available generators with `--help`. We recommend using with the `Ninja` generator, when available. More information is
 available [here](https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html).
 
+## Configuring the build on Windows
+
+There are three widely used C/C++ toolchains for Windows: *MSYS2/MinGW*, *Microsoft Visual Studio* and *LLVM/Clang*.
+Let's consider how to use each toolchain to configure and build `OMR` with `CMake`.
+
+### Common prerequisite
+
+*Perl* must be installed otherwise `CMake` won't be available to configure the project:
+
+```bash
+-- Could NOT find Perl (missing: PERL_EXECUTABLE)
+CMake Error at cmake/modules/OmrCompilerSupport.cmake:42 (message):
+  Perl not found
+```
+
+[Strawberry Perl](http://strawberryperl.com/) is absolutelly suitable for `CMake` on Windows, please do not forget
+to add the path to `perl.exe` (for example `C:\StrawberryPerl\perl\bin`) to the `PATH` environment variable.
+
+### Configuring the build for using MSYS2/MinGW
+
+The code of `OMR` uses the Microsoft extension to `C` for supporting *Windows Structured Exception Handling
```

Comment:
Is it actually true that we use SEH in OMR? Maybe somewhere in the port library? @dnakamura

Also, can we use this link instead? https://msdn.microsoft.com/library/windows/desktop/ms680657
It's the same URL, without specifying a language or VS version.

https://msdn.microsoft.com/library/windows/desktop/ms680657

Reply:
Yes, we are using SEH in the port library.